### ISSUE TITLE
App infringement

### ISSUE THREAD

**Issue by deepu2** [trebleshot/android#131]
Hi, I noticed an application in play store https://play.google.com/store/apps/details?id=com.zshare.india which copies trebleshot and has some modifications in icon and such. 
The play store description only mentions this project vaguely

```
Thanks to Genonbeta and Trebleshot.

Great thanks to Veli.

```
Moreover this app includes ads and contains in app purchases. 
I don't know if this application violates the GPL license, but I'm leaving this here for exposure.

**Comment by subins2000** [comment]
Here is another app that forked TrebleShot. It has credited, but violated GPL by not providing the source code of changes. 
https://play.google.com/store/apps/details?id=com.ayoxa.InSender

**Comment by yedhink** [comment]
Here is another app from India, which is a crystal cut copy of TrebleShot, and is injecting ads and marketing under the "MADE IN INDIA" banner.
https://play.google.com/store/apps/details?id=com.blueburn.izender

**Comment by DharwishRaj** [comment]
Is this a trend or something? This app got an article on a mainstream Newspaper in Kerala saying as Its innovative and INDIAN made!
![Screenshot_20200721-212928_Gallery](https://user-images.githubusercontent.com/28065269/88077863-60a3be00-cb99-11ea-9671-6ed6156b2b18.jpg)

**Comment by deepu2** [comment]
@DharwishRaj damn, our state has one too! The guy has claimed he had created it for his college use back when he was a student.

The new movement is creating a lot of these "Made in India" applications, not that we had enough of them already.

**Comment by nivranaitsirhc** [comment]
@velitasali I think you need to change the license of this project or modify the license so that they must explicitly ask for your permission before they publish there fork to the public.

**Comment by mubashir-rehman** [comment]
> @velitasali I think you need to change the license of this project or modify the license so that they must explicitly ask for your permission before they publish their fork to the public.

I agree with that if that not have a negative impact on the project. As far as negative impact is concerned, I am unable to see cuz those people who publish rarely open source their code.

**Comment by velitasali** [comment]
I don't think this will have a big negative impact. The only thing that the proprietary software helped us understand the importance of the community rather than the code being used in an disrespectful manner.
